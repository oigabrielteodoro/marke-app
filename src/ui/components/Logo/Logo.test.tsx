import React from 'react'
import { render, screen } from '@testing-library/react'

import { Logo } from '.'

describe('<Logo />', () => {
  it('should render correctly', () => {
    const { container } = render(<Logo />)

    expect(screen.getByRole('img')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
