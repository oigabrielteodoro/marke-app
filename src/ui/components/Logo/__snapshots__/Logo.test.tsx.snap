// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render correctly 1`] = `
.c0 {
  width: 16.4rem;
  height: 4.4rem;
}

<img
  alt="Markee App"
  class="c0"
  src="test-file-stub"
/>
`;
